### PR TITLE
feat: unify of okd and k8s tasks

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat: unify of okd and k8s tasks

this commit removes okd / kubernetes tasks manifest and keeps only a single manifest file with all tasks. This change will simplify releases and we will not have to generate two files.

**Release note**:
```
Unify okd and k8s tasks manifests

```

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -29,26 +29,15 @@ jobs:
           export RELEASE_VERSION="${{ steps.get_release.outputs.tag_name }}"
           make release
 
-      - name: Upload kubernetes manifest
-        id: kubernetes-manifest 
+      - name: Upload manifest
+        id: manifest
```
Comment (jcanocan) [inline .github/workflows/release.yaml:33]:
```suggestion
        id: manifest
```
nit: whitespace

**Review by jcanocan — COMMENTED** [review]
Just a minimal nit, apart from that, looks good.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (ksimon1) [reply .github/workflows/release.yaml]: fixed

**Review by 0xFelix — COMMENTED** [review]
Looks like `README.md` needs updates as well.

**Comment by ksimon1** [comment]
> Looks like `README.md` needs updates as well.

fixed

**Review by 0xFelix — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**